lib/common/common/Int64Math.cpp

```diff
@@ -21,7 +24,7 @@ Int64Math::Add(int64 left, int64 right, int64 *pResult)
 bool
 Int64Math::Mul(int64 left, int64 right, int64 *pResult)
 {
-#if _M_X64
+#if _M_X64 && !PLATFORM_UNIX
```

Comment:
As discussed in #192, using ```defined(_MSC_VER)``` is probably preferred. ```PLATFORM_UNIX``` is introduced for ```PAL``` and we haven't used it in ChakraCore sources yet.